lucene/suggest/src/java/org/apache/lucene/search/suggest/Lookup.java

```diff
@@ -53,7 +53,7 @@
     public final Object highlightKey;
 
     /** the key's weight */
-    public final long value;
+    public final double value;
```

Comment:
Maybe improve javadocs here, explaining that this is not just the weight originally supplied during indexing, for some suggesters?

Reply:
I agree, just adde!